src/Workspaces/Core/Portable/Storage/SQLite/v2/SQLitePersistentStorage_Helpers.cs

```diff
@@ -25,7 +23,7 @@ private static long CombineInt32ValuesToInt64(int v1, int v2)
             => ((long)v1 << 32) | (long)v2;
 
         private static (byte[] bytes, int length, bool fromPool) GetBytes(
-            Checksum checksumOpt, CancellationToken cancellationToken)
+            Checksum? checksumOpt, CancellationToken cancellationToken)
```

Comment:
consider renaming.

Reply:
We've typically been leaving `Opt`-suffix renaming for separate PRs since they are _so_ noisy in pull requests that are already large.

Reply:
I added a commit to this specifically for the RS0046 change